.github/workflows/release-macos.yml

```diff
@@ -87,6 +87,7 @@ jobs:
           7z a ${SLANG_BINARY_ARCHIVE} slang-gfx.h
           7z a ${SLANG_BINARY_ARCHIVE} prelude/*.h
           7z a ${SLANG_BINARY_ARCHIVE} bin/*/*/libslang.dylib
+          7z a ${SLANG_BINARY_ARCHIVE} bin/*/*/libslang-glslang.dylib
```

Comment:
Need to codesign and notarize this dylib too.

Reply:
Done. Also added `libslang-glslang.dylib` to the dist package.